imagebuildah/build.go

```diff
@@ -1510,6 +1522,45 @@ func (b *Executor) Build(ctx context.Context, stages imagebuilder.Stages) (image
 	}
 	defer cleanup()
 
+	// Build maps of every named base image and every referenced stage root
+	// filesystem.  Individual stages can use them to determine whether or
+	// not they can skip certain steps near the end of their stages.
+	for _, stage := range stages {
+		node := stage.Node
+		for node != nil {
+			for _, node := range node.Children {
```

Comment:
Can we rename the loop's run variable to `child`? The code is correct but it looked very suspicious, so I had parse it a few times to be sure to understand it's correct.

Reply:
or if you really don't like child, some other variable name, or perhaps change node to stageNode at line 1529

Reply:
`child` is fine.  Changed it.